### ISSUE TITLE
add docs section on context handling

### ISSUE THREAD

**Issue by amn41** [RasaHQ/rasa_nlu#951]
link to Rasa Core from NLU docs

**Comment by wrathagom** [comment]
@amn41 if you're looking for a community contrib on this one may need more info. Are you just asking for a page titles "Adding Dialog" with a blurb and link to Core?

**Comment by amn41** [comment]
yes - sorry this was more of a note to self, but it's exactly as you describe :)

**Comment by amn41** [comment]
@ricwo can you please create a PR for this?
